java-manta-cli/pom.xml

```diff
@@ -17,6 +17,10 @@
     <artifactId>java-manta-cli</artifactId>
     <packaging>jar</packaging>
 
+    <properties>
+        <dependency.picocli.version>0.9.8</dependency.picocli.version>
```

Comment:
Even if it's only relevant to the cli module I'd prefer we keep all version properties in the parent pom. I noticed the benchmark module does this for JMH as well and am not asking you to clean those up, just to move this one to the parent pom if possible.